bears/general/SpaceConsistencyBear.py

```diff
@@ -38,7 +41,52 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
-        for line_number, line in enumerate(file, start=1):
+        def get_blanklines_nr_start():
+            line_nr_start = False
+            if not allow_trailing_blanklines:
+                rev_enumerated_zip_obj = zip(reversed(range(1, len(file) + 1)),
+                                             reversed(file))
+                rev_enumerated_tuple = tuple(rev_enumerated_zip_obj)
+
+                for line_number, line in rev_enumerated_tuple:
+                    replacement = line
+                    if replacement.strip() == '':
+                        line_nr_start = line_number
+                    else:
+                        break
+
+            return line_nr_start
+
+        if allow_trailing_blanklines:
```

Comment:
I think there should be a `not` in here. Please correct me if I am wrong

Reply:
No.If trailing blanklines are allowed than then we don't to do any analysis for checking blanklines.